src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -41,22 +55,27 @@
   private static final String START_ATTRIBUTE = "start";
   private static final String DURATION_ATTRIBUTE = "dur";
```

Comment:
nit: could you rename these to ATTR_START and ATTR_DURATION?

Reply:
this way it's easy to see a block of attributes, a block of params, etc.

Reply:
Update:  I changed it to be ATTR_START and ATTR_DURATION to be consistent with the other files.